### ISSUE TITLE
bluedevil kcmshell modules missing

### ISSUE THREAD

**Issue by schmittlauch** [NixOS/nixpkgs#47438]
## Issue description

Some important kcmshell (KDE configuration tool) modules for administrating bluetooth devices are apparently missing:

- bluedeviladapters
- bluedevilglobal
- bluedevildevices

### Steps to reproduce

When clicking on the settings icon in the bluetooth plasmoid (from the system tray), no window opens but an error message appears in the user's journal.

1. precondition: a NixOS 18.03 installation with KDE plasma and enabled bluetooth
1. open the KDE plasma bluetooth plasmoid form the systray
1. click on the settings icon in the top right corner
1. see an error message appear in the systemd user journal:

```
xsession[1234]: Das Modul bluedevildevices wurde nicht gefunden. Der Befehl „kcmshell5 --list“ zeigt eine vollständige Liste der Module an.
xsession[1234]: Das Modul bluedeviladapters wurde nicht gefunden. Der Befehl „kcmshell5 --list“ zeigt eine vollständige Liste der Module an.
xsession[1234]: Das Modul bluedevilglobal wurde nicht gefunden. Der Befehl „kcmshell5 --list“ zeigt eine vollständige Liste der Module an.
```

rough translation into English:

```
xsession[1234]: The module bluedevildevices could not be found. The command „kcmshell5 --list“ displays a complete list of modules
```

and the same with the two other missing modules. All mentioned modules do *not* appear under `kcmshell5 --list`.

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.71, NixOS, 18.03.133245.d16a7abceb7 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.4`
 - channels(root): `"nixos-18.03.133245.d16a7abceb7, nixos-unstable-19.03pre153673.a70d3bab72d"`
 - channels(spiollinux): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by aanderse** [comment]
@schmittlauch is this fixed for you in 18.09? The bluetooth kcm shows up fine for me.

**Comment by schmittlauch** [comment]
@aanderse It indeed is fixed now, thanks for the reminder.
